src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -546,6 +546,22 @@ public function fill(array $attributes)
         return $this;
     }
 
+    /**
+     * Fill only the missing with an array attributes of the model.
+     *
+     * @param  array  $attributes
+     * @return $this
+     */
+    public function fillMissing(array $attributes)
```

Comment:
Maybe `fillBlanks` is a better name as we have the filled() & blank() helpers, this way we can follow a "convention"

Reply:
Thanks, I updated the name to `fillBlanks`